web/src/features/auth/slices/authenticationSlice.test.ts

```diff
@@ -159,6 +167,7 @@ describe('authenticationSlice', () => {
           token: testToken,
           idToken: testToken,
           idir: 'test@idir',
+          email: 'test@example.com',
```

Comment:
nit: Can this use the `email` variable you defined on line 27?

Reply:
Thanks fixed in https://github.com/bcgov/wps/pull/3658/commits/3a3c42d89187ad7d5ac0f2d8aa6aa3c3c5902465